src/design-guidelines/action-buttons/action-buttons.md

```diff
@@ -0,0 +1,126 @@
+
+<br/>
+
+*****
+
+## Choosing good labels
+
+Labels you use should make sense together. You are usually dealing with contradictory actions. Therefore, what you put on each button should also reflect the contradiction.
+
+<limel-example-action-buttons-choosing-labels></limel-example-action-buttons-choosing-labels>
+
+Labels must be making each option as clear as possible. An explicit label give users more confidence in selecting the correct action.
+
+:::note
+- Make sure the label explains what the button does, rather than a generic label (such as “OK”, or "Yes").
+- Using verbs as labels instead of nouns is better, because a verb describes the action better.
+:::
+
+<limel-example-action-buttons-choosing-explicit-labels></limel-example-action-buttons-choosing-explicit-labels>
+
+<br/>
+
+*****
+
+## Providing a third alternative action
+
+Sometimes having two action is not enough for the user to take a decision with peace of mind. A common scenario is when a user has unsaved changes and tries to navigate away from the view. In such cases, typically a prompt maybe be shown to alert the user and give them a few logical choices.
+
+This prompt can offer two main actions of "Save" and "Discard" (which means "save & continue navigating away from this page" and "discard & continue navigating away from this page"), along with a third choice that allows users to "Cancel" (and go back / not continue navigating away!).
+
+<limel-example-action-buttons-third-alternative></limel-example-action-buttons-third-alternative>
+
+The action that takes users one step back inte the process is intentionally placed on the left side. This firstly makes it easier for users to mentally separate the other two, and secondly the placement makes it easier to connect the action to the notion of going back, in a UI which is designed for left-to-right script.
+
+:::tip
+If buttons are placed in a full width flex container that has `justify-content: flex-end;`, these styles will align the targeted button to left side of the flex box. See more [in examples of `limel-dialog`](#/components/limel-dialog).
+
+```scss
+.back-button {
+    justify-self: flex-start;
+    margin-right: auto;
+}
+```
+:::
+
+<br/>
+
+*****
+
+## Adding more meaning with colors and icons
+
+To make it easer for users to understand the message and take a decision faster, you can use icons and colors on the buttons.
+
+Learn more about [usage of colors and color conventions in the UI](#/DesignGuidelines/color-system.md/), and [how to color limel-buttons](#/component/limel-button/).
+
+<limel-example-action-buttons-icon-color></limel-example-action-buttons-icon-color>
+
+:::note
+It might be tempting to use colors on action buttons, with the intention of making the user interface more cheerful. But using colors in a user interface must follow certain UI design conventions, as they convey meanings. Learn more about [usage of colors and color conventions in the UI](#/DesignGuidelines/color-system.md/).
```

Comment:
```suggestion
It might be tempting to use colors on action buttons to make the user interface more cheerful. But using colors in a user interface must follow certain UI design conventions, as colors convey meaning. Learn more about [usage of colors and color conventions in the UI](#/DesignGuidelines/color-system.md/).
```